client/header/activity-panel/index.js

```diff
@@ -208,7 +212,15 @@ export class ActivityPanel extends Component {
 			  }
 			: null;
 
-		return [ inbox, ...stockAndReviews, setup, help ].filter( Boolean );
+		const displayOption = showDisplayOptions
```

Comment:
Should be named `displayOptions`

Reply:
Updated :) thanks for noticing. Updating status to ready for merge (merging when CI works again)